site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Getting the NGINX Gateway Fabric Image with JWT
```

Comment:
We should probably be specific in these docs that this is only for pulling the NGINX Plus NGF image.

Reply:
updated